client/pages/forum/forum.tsx

```diff
@@ -35,21 +38,59 @@ const data = [
   },
 ];
 
+const pagesCount = 5;
+
 const Forum: FC<Props> = ({ className }) => {
-  /* right now it all is just a test of the table component */
   const [sort, handleSortRequest] = useNullableTableSort();
+  const [page, setPage] = useState(1);
+
+  const handleCreateTopicClick = useCallback(() => {
+    alert("need to create a new topic!");
+  }, []);
+
+  const handlePageClick = useCallback((newPage: number) => {
+    setPage(newPage);
+  }, []);
+
+  const handleGoToFirstPageClick = useCallback(() => {
+    setPage(1);
+  }, []);
+
+  const handleGoToLastPageClick = useCallback(() => {
+    setPage(pagesCount);
+  }, []);
```

Comment:
```suggestion
  }, [pagesCount]);
```

Reply:
Это внешний prop, он должен быть в зависимостях хука.
Добавьте правило `eslint(react-hooks/exhaustive-deps)` если его еще нет.

Reply:
В данном случае, всё-таки, не нужно:

ESLint: React Hook useCallback has an unnecessary dependency: 'pagesCount'. Either exclude it or remove the dependency array. Outer scope values like 'pagesCount' aren't valid dependencies because mutating them doesn't re-render the component.(react-hooks/exhaustive-deps)

Reply:
После подключения api - да, придётся добавить. А пока это только константа - видимо, нет.